### ISSUE TITLE
Error if you inadvertently add test to itself

### ISSUE THREAD

**Issue by alecmce** [lukebayes/asunit#5]
I found that I had inadvertently added a test suite to itself today, but got an error that wasn't terribly helpful in working out what I had done. It would be good if ASUnit handled this without going to a recursion stack overflow.

(I'll do it myself if I remember, but wanted to add the issue here in case I forgot)

Cheers,

Alec

**Comment by robertpenner** [comment]
Do you mean a test suite?

**Comment by alecmce** [comment]
Yes, sorry, I meant a test suite. I said so in the body, but not in the title. I haven't looked into it but imaigne it's a pretty trivial fix; if you're refactoring then I'll wait until your repository is pulled then I'll have a look at making the fix.

**Comment by robertpenner** [comment]
I haven't done much in TestCase or TestSuite. You can pull my changes from master yourself, make the change, and then Luke can pull from you. With Git, the order is more flexible.
